guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
 
 # Sending bitcoin
 
-Sending bitcoin can be a very straightforward or complex flow in a Bitcoin application. People may be sending bitcoin to a known contact, moving it between their wallets on different devices, or making a purchase through a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
+Sending bitcoin can be a very simple, or complex flow in a Bitcoin application. People may want to send bitcoin to a known contact, transfer it to another wallet on a different device, or make a purchase via a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
 
-Let us look at the main transaction options senders need to configure when moving bitcoin:
+## Payment entry points
 
-- **Amount** — How much to send
-- **Recipient address** — Where to send the bitcoin
-- **Coin selection** — Which coins/inputs to use (optional)
-- **Fee settings** — Prioritize fast confirmation or low cost (optional)
+The need to send bitcoin can be triggered by many different use cases, and initiated by both sender and receiver. Common entry points are:
 
-You do not need to follow the order below. Feel free to tailor the configuration's order for the payment to what best serves your users. For example, you may make users set the amount before they enter the address.
+- Manual entry of address and amount
+- Sending to a previously saved contact
+- Copy & pasting payment information
+- Clicking a payment link
+- Scanning a QR code
+- Holding a device close to another device (using [NFC](https://en.wikipedia.org/wiki/Near-field_communication))
```

Comment:
```suggestion
- Receiving a payment request over ([NFC](https://en.wikipedia.org/wiki/Near-field_communication))
```

Reply:
Should this also include BlueTooth for completeness?

Reply:
Could just say something like 'Receive a payment request with a contact less payment (NFC or BlueTooth).